quantization_troubleshooting.md

```diff
@@ -0,0 +1,153 @@
+# Quantization Troubleshooting for MCT
+
+**Compressing a model with MCT:**
```

Comment:
Rephrasing suggestion for the intrduction (yes, I consulted chatgpt it's not totally my phrasing):

**Utilizing MCT for Model Compression:**
The Model Compression Toolkit (MCT) offers numerous functionalities to compress neural networks with minimal accuracy lost. 
However, in some cases, the compressed model may experience a significant decrease in accuracy. 
Fear not, as this lost accuracy can often be reclaimed by adjusting the quantization configuration or setup.

**Restoring Accuracy - Troubleshooting Steps:**
Outlined below are a series of steps aimed at recovering lost accuracy resulting from compression with MCT. 
Some steps may be applicable to your model, while others may not.